<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TorClientWindow</class>
 <widget class="QMainWindow" name="TorClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>474</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>65535</width>
    <height>65535</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tor Client</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/onion.png</normaloff>:/images/onion.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1" colspan="2">
     <widget class="QTextBrowser" name="logBrowser"/>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Socks Port</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSpinBox" name="controlPortSpinBox">
      <property name="minimum">
       <number>1024</number>
      </property>
      <property name="maximum">
       <number>5050</number>
      </property>
      <property name="value">
       <number>5050</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2">
     <widget class="QSpinBox" name="socksPortSpinBox">
      <property name="minimum">
       <number>1024</number>
      </property>
      <property name="maximum">
       <number>65535</number>
      </property>
      <property name="value">
       <number>9050</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Control Port</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>F&amp;ile</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="actionNewIdentity"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionStart">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/switch_off.png</normaloff>
     <normalon>:/images/switch_on.png</normalon>:/images/switch_off.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string>Start Tor Client</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNewIdentity">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/employee_card.png</normaloff>:/images/employee_card.png</iconset>
   </property>
   <property name="text">
    <string>NewIdentity</string>
   </property>
   <property name="toolTip">
    <string>New Identity</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
